cartridges/int_algolia_sfra/cartridge/templates/default/search/searchResults.isml

```diff
@@ -4,6 +4,7 @@
         var assets = require('*/cartridge/scripts/assets');
         assets.addJs('/js/search.js');
         assets.addCss('/css/search.css');
+        assets.addJs('/js/utils/productTileUtils.js');
```

Comment:
Is the bug reproducible on search results pages too? This code is copied from SFRA, is there a bug in the base cartridge?

Reply:
no, we are not including productDetail.js here and it is not causing any issue. It is not a bug that is coming from base cartridge

Reply:
Ah, it's to fix the add to cart button that is disabled then I guess. Ok :+1: